tests/foreman/ui/test_ansible.py

```diff
@@ -229,6 +236,27 @@ def _finalize():
                 (v['Name'], v['Ansible role'], v['Type'], v['Value']) for v in variable_table
             ]
 
+            new_key = gen_string('alpha')
+            session.ansiblevariables.create_with_overrides(
+                {
+                    'key': new_key,
+                    'ansible_role': SELECTED_ROLE[0],
+                    'override': 'true',
+                    'parameter_type': parameter_type,
+                    'default_value': default_value,
+                }
+            )
+            new_value = '["test update"]'
```

Comment:
Doesn't this give you error?
You can try new_value = '[\"test update\"]'

Reply:
It did not produce any errors, but I made a small change.